src/Servers/Kestrel/test/InMemory.FunctionalTests/EventSourceTests.cs

```diff
@@ -423,7 +423,7 @@ public async Task ConnectionLimitExceeded_EmitsStartAndStopEventsWithActivityIds
             Assert.Equal(eventIndex, events.Count);
         }
```

Comment:
Your choice in deciding how much you care, but if you want to you could write a unit test that deserializes the serialized JSON to see if the value round trips. Another option would be reflecting over all property names to confirm that they match the names observed in the JSON.

Reply:
I will write a test that JSON deserializes